### ISSUE TITLE
integration test harness

### ISSUE THREAD

**Issue by flihp** [tpm2-software/tpm2-tcti-uefi#22]
Running the example UEFI applications under Qemu with OVMF is super handy (see #21) . The remaining tasks to turn this approach into a proper integration test harness should be considered / discussed here.

**Comment by johnandersen777** [comment]
I'm working on getting the travis build matrix working again now. We could start with just running the one binary in CI after the tests to ensure its working. I have a grep check to make sure get caps gets an RC_SUCCESS right now.

Perhaps the `startup.nsh` could run each binary and then we grep the ouputs of each to determine success?

> Edit: Yes, this is a pretty hacky approach

##### example

```bash
timeout 30s unbuffer sh lib/qemu-tpm2.sh | tee qemu_example.log
exec grep -q 'Tss2_Sys_GetCapability returned TSS2_RC_SUCCESS' qemu_example.log
```

**Comment by johnandersen777** [comment]
I was able to re-enable the build matrix and get one efi script running in travis after the build and unit tests #23

**Comment by flihp** [comment]
resolved by #28
